blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphVisitor.java

```diff
@@ -311,18 +317,18 @@ public void parallelStart(@Nonnull FlowNode parallelStartNode, @Nonnull FlowNode
                                branchNode.getDisplayName(), branchNode.getDisplayFunctionName()));
         }
 
-        if ( nestedbranches.size() != parallelBranchEndNodes.size() ) {
+        if (nestedbranches.size() != parallelBranchEndNodes.size()) {
             logger.debug(String.format("nestedBranches size: %s not equal to parallelBranchEndNodes: %s",
-                nestedbranches.size(), parallelBranchEndNodes.size()));
-            if(!parallelEnds.isEmpty()){
+                                       nestedbranches.size(), parallelBranchEndNodes.size()));
```

Comment:
the indentation looks a tad off here

Reply:
Nah, those are args to `String.format()` not `logger.debug()`